File: LeetCode Solutions/Problems/2190. Most Frequent Number Following Key In an Array.py
class Solution:
    def mostFrequent(self, nums: List[int], key: int) -> int:
        count = Counter()
        for i in range(len(nums)):
            if nums[i] == key and i < len(nums) -1:
                count[nums[i+1]] += 1
        return count.most_common()[0][0]

File: LeetCode Solutions/Problems/2190. Most Frequent Number Following Key In an Array.cpp
class Solution {
public:
    int mostFrequent(vector<int>& nums, int key) {
        int count[1001] = {}, result = 0;
        for (int i=1; i<nums.size(); ++i){
            if (nums[i - 1] == key && ++count[nums[i]] > count[result]){
                result = nums[i];
            }
        }
        return result;
    }
};